clarifai/client/app.py

```
@@ -288,6 +288,42 @@ def list_concepts(self, page_no: int = None,
       concept_info['id'] = concept_info.pop('concept_id')
       yield Concept(**concept_info)
 
+  def list_concept_relations(self,
+                             concept_id: str = None,
+                             predicate: str = None,
```

```suggestion
                             predicate: str = '',
```

---

Both None or empty string are working to get all the concept relations of a given concept

---

Okay, then we can just make it optional, letting the users know that by default it returns all the relations.

---

In this case add logging, if the request fails, We should return the error logging message in case user gives a different predicate in the request.

---

Logging already present in the lister class

---

Great then everything looks fine.